stdlib/public/TensorFlow/Gradients.swift

```diff
@@ -301,13 +300,13 @@ extension Tensor where Scalar : Differentiable & FloatingPoint,
     lhs: Scalar, rhs: Tensor
   ) -> (Tensor, (Tensor) -> (Scalar, Tensor)) {
     return (lhs / rhs, { v in
-      ((v / rhs).sum().scalarized(), v * (0 - lhs) / rhs.squared())
```

Comment:
This seems fine as long as you know why it was `0 - lhs` before and the reason no longer applies.

Reply:
This is because `Scalar` was not constraint on `FloatingPoint` conformance (but `Numeric`), which was because `@differentiable` did not support a `where` clause :)